src/Illuminate/Foundation/Http/Kernel.php

```diff
@@ -214,6 +214,8 @@ public function terminate($request, $response)
 
         $this->app->terminate();
 
+        $this->requestStartedAt->setTimezone($this->app['config']->get('app.timezone') ?? 'UTC');
```

Comment:
@timacdonald 
this
 `get('app.timezone') ?? 'UTC')`
 Shouldn't it be like this?
`get('app.timezone', 'UTC')`

Reply:
@antonkomarev 
So, this means 
```php
$this->commandStartedAt->setTimezone($this->app['config']->get('app.timezone', 'UTC'));
``` 
should be changed to 
```php
$this->commandStartedAt->setTimezone($this->app['config']->get('app.timezone') ?? 'UTC');
``` 
in `console` kernel?

Reply:
I think this PR created an issue with Vapor :/
https://flareapp.io/share/yPa2l9K7

https://github.com/laravel/framework/issues/47591

Reply:
Fix: https://github.com/laravel/framework/pull/47596